src/components/Navbar.tsx

```diff
@@ -15,12 +15,22 @@ const Navbar = () => {
           history.push('/')
         },
       }}
-      onSearchButtonClick={() => console.log('hello')}
-      onSearchTextBoxChange={() => console.log('hello')}
+      search={{
+        onClickButton: () => {
+          console.log('search')
+        },
+        onChangeInput: () => {
+          console.log('change')
+        },
+        placeholderText: t('actions.search'),
+        buttonText: t('actions.search'),
+      }}
       navLinks={[
         {
           label: t('patients.label'),
-          onClick: () => {},
+          onClick: () => {
+            console.log('patients click')
```

Comment:
these `console.log()` are only here to appease the linter. I've opened an issue to make them optional: https://github.com/HospitalRun/components/issues/178